tools/demos/install/demo.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2016-2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. $(dirname ${BASH_SOURCE})/../util.sh
+
+desc "Let's install Inspektor Gadget"
+run "kubectl gadget deploy"
+
+run "kubectl get pod -n gadget"
+
+desc "And check the installed version"
+run "kubectl logs -n gadget -l k8s-app=gadget --tail=-1 | grep 'Inspektor Gadget version'"
```

Comment:
nit (for another PR): It should be handled by "kubectl gadget version" as kubectl does -> https://github.com/kinvolk/inspektor-gadget/issues/902.